### PR TITLE
Ensure non-parented names are included in output

### PR DESCRIPTION
**Pull request by charvolant**
Allow ' X ' hybrids against God's law
Fix for issue #12

### CODE REVIEW

File: ala/transform.py
Comment (Sherrin-ALA) [inline ala/transform.py:1]: Is there a change for handling vernacular names in the species lists in this update as well?

File: dwc/transform.py
Comment (Sherrin-ALA) [inline dwc/transform.py:1]: Is the only change here adding X to the regex for HYBRID names (line 611)?

**Review by Sherrin-ALA — APPROVED** [review]
Looks good - questions are just really for my own information.

File: ala/transform.py
Comment (charvolant) [reply ala/transform.py:1]: Vernacular names issue appears to be a problem with loading lists into the lists tool

File: dwc/transform.py
Comment (charvolant) [reply dwc/transform.py:1]: That and making sure that something with no parents still gets through.

**Pull request merged**